### PR TITLE
Implement styling for navigation, home and aboutus page

### PR DESCRIPTION
**Pull request by anqitu**

### CODE REVIEW

File: src/main/webapp/aboutus.html [hunk 1]
```diff
@@ -17,79 +17,24 @@
...
-        back pain and shoulder pain due to sitting in front of the screen for
-        almost entire day. Doing yoga also reminds me to leave some time to slow
-        down the pace of my life and enjoy the current moment.
-      </li>
-    </ul>
+  <body onload="buildUI()">
+    <div class="text-center page-header">
+      <h1 class="display-4">Meet the Team</h1>
+      <p class="font-weight-lighter"
+        >It's such an coincidence that we all start our name as <bold class="font-weight-bold">A</bold>.
```
Comment (amad-person) [inline src/main/webapp/aboutus.html]:
```suggestion
        >It's such a coincidence that all our names start with an <bold class="font-weight-bold">A</bold>.
```

File: src/main/webapp/css/navigation-bar.css [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+  background-color: rgba(255, 255, 255, 0) !important;
+  border-bottom: 1px solid rgba(207, 203, 203, 0.4);
 }
 
-/* Keep navigation menu on one line. */
-nav li {
-  display: inline-block;
-  margin-right: 10px
+nav.navbar {
+  /* background-color: rgba(95, 103, 112, 1); */
```
Comment (amad-person) [inline src/main/webapp/css/navigation-bar.css]: Was this supposed to be uncommented? Just confirming

**Review by amad-person — COMMENTED** [review]
Thanks for the PR @anqitu it looks great! Just some minor things that need to be addressed and then we can merge this

File: src/main/webapp/css/navigation-bar.css (hunk 2 above)
Comment (anqitu) [reply src/main/webapp/css/navigation-bar.css]: Oh yes supposed to be deleted.

**Review by amad-person — APPROVED** [review]
LGTM

**Pull request merged**